### ISSUE TITLE
Native App Issue: Messages send and display component

### ISSUE THREAD

**Issue by abhijeet007rocks8** [DhairyaBahl/React-Messenger-App#78]
This component will render on successful login.
The layout will be similar to that of the web version-:  Refer <a href="mui-message.firebaseapp.com">Website</a>
Additional Feature-: the sender profile pic will be visible beside the message.  

(Do not Focus on CSS too much in earlier phases)

**Comment by kunal-j10** [comment]
@abhijeet007rocks8 I am open to working on this.

**Comment by abhijeet007rocks8** [comment]
Go Ahead @kunal-j10 
Fork the Native-Branch and push updates to it

**Comment by abhijeet007rocks8** [comment]
@kunal-j10 any updates??

**Comment by anshika208** [comment]
hey! @abhijeet007rocks8 if the issue is still open and there are no updates from assignee then may i work on this issue?

**Comment by abhijeet007rocks8** [comment]
@kunal-j10 Please respond if you are still working on the issue.
By the time @anshika208 you can start working on this issue.
Advice-: you would have to use your personal firebase keys while development.

**Comment by abhijeet007rocks8** [comment]
@anshika208 any updates on the issue??

**Comment by anshika208** [comment]
Yes i m still working but most probably will make a pr till tomorrow

**Comment by abhijeet007rocks8** [comment]
@anshika208 Any updates on the issue??
If you are facing any issue feel free to discuss it with mentors

**Comment by abhijeet007rocks8** [comment]
@anshika208 any updates??

**Comment by subhangi2731** [comment]
@abhijeet007rocks8 please assign this issue to me

**Comment by subhangi2731** [comment]
@abhijeet007rocks8 please assign this to me
